business/health.go

```diff
@@ -333,3 +338,10 @@ func castWorkloadStatuses(ws models.Workloads) []models.WorkloadStatus {
 	}
 	return statuses
 }
+
+func generateThreshold(requests model.Vector, ns string, srv string, kind string, labels map[string]string) models.Thresholds {
+	thAlerts := threshold.FilterBy(config.Get().ThresholdsHealth, ns, srv, kind, labels)
+	thresholds := models.Thresholds{}
+	thresholds.Parse(thAlerts, &requests, "service")
```

Comment:
I guess "service" should be `kind` here?

Reply:
Yes this was last commit. I forgot it sorry